openlibrary/utils/__init__.py

```diff
@@ -212,7 +212,7 @@ def get_software_version() -> str:
     assert get_software_version()  # Should never return a falsy value
     """
     cmd = "git rev-parse --short HEAD --".split()
-    return run(cmd, capture_output=True, text=True).stdout.strip()
+    return run(cmd, capture_output=True, text=True, check=True).stdout.strip()
```

Comment:
What is this supposed to do?  The `check=True` is causing [this](https://github.com/internetarchive/openlibrary/issues/8211) issue.

Reply:
% `ruff rule W1510`
https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/subprocess-run-check.html

Reply:
It's definitely breaking local environments.  Staging seems to be in good shape, though.

Reply:
Is if possible to get the full traceback on the failures?